core/src/identity/rsa.rs

```diff
@@ -107,6 +107,24 @@ impl PublicKey {
     }
 }
 
+fn to_hex(bytes: &[u8]) -> String {
```

Comment:
This is very nit-picky, but I'd move the body of `to_hex` to within the `fmt` function, because it's used only there.

Reply:
I don't mind nit-pickyness at all. Will do.